src/winkerberos.c

```diff
@@ -775,6 +785,12 @@ initwinkerberos(VOID)
                            "GSS_C_INTEG_FLAG",
                            PyInt_FromLong(ISC_REQ_INTEGRITY)) ||
         PyModule_AddObject(module,
+                           "GSS_MECH_OID_KRB5",
+                           PyCapsule_New(GSS_MECH_OID_KRB5, "winkerberos.GSS_MECH_OID_KRB5", NULL)) ||
```

Comment:
We have to support Python versions that don't provide PyCapsule. I've aliased the necessary PyCObject* functions to the PyCapsule equivalents at the top of this file. Change this to:

PyCObject_FromVoidPtr(GSS_MECH_OID_KRB5, NULL)